ui/user/src/lib/components/tasks/Step.svelte

```diff
@@ -110,7 +110,7 @@
 			class="flex-1 resize-none border-none bg-gray-50 outline-none dark:bg-gray-950"
 		></textarea>
 		<div class="flex gap-2 p-2">
-			<button class="rounded-lg p-2 hover:bg-gray-50 dark:hover:bg-gray-950" onclick={doRun}>
+			<button class="icon-button-small" onclick={doRun}>
```

Comment:
ivys revamping tasks. maybe better to not touch it so as to not create a conflict for her?